src/canvas/textFeature.js

```diff
@@ -0,0 +1,214 @@
+var inherit = require('../inherit');
+var registerFeature = require('../registry').registerFeature;
+var textFeature = require('../textFeature');
+var util = require('../util');
+var mat3 = require('gl-mat3');
+var vec3 = require('gl-vec3');
+
+/**
+ * Create a new instance of class canvas.textFeature.
+ *
+ * @class
+ * @alias geo.canvas.textFeature
+ * @extends geo.textFeature
+ * @extends geo.canvas.object
+ *
+ * @param {geo.textFeature.spec} [arg] Options for the feature.
+ * @returns {geo.canvas.textFeature} The created feature.
+ */
+var canvas_textFeature = function (arg) {
+  'use strict';
+  if (!(this instanceof canvas_textFeature)) {
+    return new canvas_textFeature(arg);
+  }
+
+  var object = require('./object');
+
+  arg = arg || {};
+  textFeature.call(this, arg);
+  object.call(this);
+
+  /**
+   * @private
+   */
+  var m_this = this,
+      m_defaultFont = 'bold 16px sans-serif',
```

Comment:
I picked `bold 16px sans-serif` because I thought it was a good default for the annotations.  Generally, I think a text feature will be used for labels not blocks of text, so a larger size is more likely to be appropriate.  I could be persuaded otherwise.